pom.xml

```diff
@@ -77,6 +77,7 @@
         <module>plugin/trino-geospatial</module>
         <module>plugin/trino-google-sheets</module>
         <module>plugin/trino-hive</module>
+        <module>plugin/trino-hsqldb</module>
```

Comment:
When adding a new connector, it's necessary to add it to `core/trino-server/src/main/provisio/trino.xml` to make it part of the build.

If there was a product test (see `testing/trino-product-tests/README.md`), it would verify this works correctly in real deployment

Reply:
> When adding a new connector, it's necessary to add it to core/trino-server/src/main/provisio/trino.xml to make it part of the build.
I just saw this and made the necessary modification in order to have the possibility of testing this new connector.  

And I still have a connection problem to the hsqldb docker container that I have to fix.